bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/update/thing-updates.xml

```diff
@@ -36,6 +36,13 @@
 				<description>The average experience of the connected clients</description>
 			</add-channel>
 		</instruction-set>
+		<instruction-set targetVersion="2">
+			<add-channel id="led">
+				<type>system:power</type>
+				<label>LED</label>
+				<description>Switches the LED on or off</description>
```

Comment:
Setting label and description should not be needed for new added channels. It won't harm, but just saying.